cmd/podman/image.go

```diff
@@ -3,6 +3,7 @@ package main
 import (
 	"github.com/containers/libpod/cmd/podman/cliconfig"
 	"github.com/spf13/cobra"
+	"strings"
```

Comment:
The proper format is to have 
golang imports separated by a space from github.com and others
import (
	"strings"

	"github.com/containers/libpod/cmd/podman/cliconfig"
	"github.com/spf13/cobra"
)

Reply:
@rhatdan thanks for the correction. Fixed and force-pushed.